pdk.gemspec

```diff
@@ -17,28 +17,27 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.4.0' # rubocop:disable Gemspec/RequiredRubyVersion
+  spec.required_ruby_version = '>= 2.5.9' # rubocop:disable Gemspec/RequiredRubyVersion
 
-  spec.add_runtime_dependency 'bundler', '>= 1.15.0', '< 3.0.0'
-  spec.add_runtime_dependency 'childprocess', '~> 4.0.0'
-  spec.add_runtime_dependency 'cri', '~> 2.10'
-  spec.add_runtime_dependency 'diff-lcs', '>=1.4.4', '< 1.5'
-  spec.add_runtime_dependency 'ffi', '>= 1.9.25', '< 2.0.0'
+  spec.add_runtime_dependency 'bundler', '>= 2.3.0', '< 3.0.0'
+  spec.add_runtime_dependency 'childprocess', '~> 4.1.0'
+  spec.add_runtime_dependency 'cri', '~> 2.15.11'
+  spec.add_runtime_dependency 'diff-lcs', '>= 1.5.0'
+  spec.add_runtime_dependency 'ffi', '>= 1.15.5', '< 2.0.0'
   spec.add_runtime_dependency 'hitimes', '2.0.0'
   spec.add_runtime_dependency 'json-schema', '2.8.0'
-  spec.add_runtime_dependency 'json_pure', '~> 2.1.0' if RUBY_VERSION < '2.7'
-  spec.add_runtime_dependency 'json_pure', '~> 2.5.1' if RUBY_VERSION >= '2.7'
+  spec.add_runtime_dependency 'json_pure', '~> 2.6.2'
```

Comment:
will ruby versions always be >= 2.7 now?

Reply:
No. json_pure 2.6.2 has a requirement for Ruby >= 2.3. Our min ruby is now set to 2.5.9 so we are good to have the single dependency line.

Reply:
great